bindings/matlab/iDynTreeMATLAB_wrap.cxx

```diff
@@ -1,6 +1,6 @@
 /* ----------------------------------------------------------------------------
  * This file was automatically generated by SWIG (http://www.swig.org).
- * Version 3.0.9
+ * Version 3.0.5
```

Comment:
Sorry, I hadn't compiled properly SWIG after selecting the correct commit. Now it's SWIG build has been done, bindings regenerated and pushed again.